### ISSUE TITLE
Update `Tensor.sum`

### ISSUE THREAD

**Issue by KOHbDS** [haydnv/tinychain#88]
Currently, I use `Tensor.sum()`, and I found that `sum(axis={0, 1})` works only two dimensions. To my mind, this method has to work for all dimensions, it looks like:
```python
Tensor.sum(axis=[0, 2, 3])
```
ref:https://pytorch.org/docs/stable/generated/torch.sum.html

**Comment by haydnv** [comment]
As a workaround you can use `x.sum(0).sum(1).sum(1)` or possibly even `einsum("ijklm->jm", [x])` but I do like the `sum([1, 2, 3])` syntax better

**Comment by haydnv** [comment]
This issue is superceded by #167 167
